qwandaq/src/main/java/life/genny/qwandaq/utils/DefUtils.java

```diff
@@ -240,28 +249,29 @@ public Boolean answerValidForDEF(Definition definition, Answer answer) {
 	 */
 	public Boolean attributeValueValidForDEF(BaseEntity defBE, AttributeCodeValueString acvs) {
```

Comment:
Gonna squash this weird class like a bug at some stage (AttributeCodeValueString)